src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -628,9 +635,9 @@ def __print_features_preview(old_json, new_json):
     return True
 
 
-def __print_preview(old_json, new_json):
+def __print_preview(old_json, new_json, strict):
```

Comment:
>strict [](http://example.com/codeflow?start=40&length=6)

need a default value for strict argument because this method is invoked during export operation too.